core/foundation/inc/ROOT/StringUtils.hxx

```diff
@@ -23,6 +23,25 @@ namespace ROOT {
 
 std::vector<std::string> Split(std::string_view str, std::string_view delims, bool skipEmpty = false);
 
+std::string Join(const std::string& sep, std::vector<std::string> strings);
+
+/**
+ * @brief Concatenate several std::string
```

Comment:
Also one note on the side, ROOT doxygen prefers the `\brief` syntax over the `@brief` syntax (similarly for all other doxygen tags). Could you also modify the docs?